Authenticator/Source/TokenEditForm.swift

```diff
@@ -58,21 +58,15 @@ class TokenEditForm: TokenForm {
     var viewModel: TableViewModel<Form> {
         return TableViewModel(
             title: "Edit Token",
-            leftBarButton: BarButtonViewModel(style: .Cancel) { [weak self] in
-                self?.cancel()
-            },
-            rightBarButton: BarButtonViewModel(style: .Done, enabled: state.isValid) { [weak self] in
-                self?.submit()
-            },
+            leftBarButton: BarButtonViewModel(style: .Cancel, action: .Cancel),
+            rightBarButton: BarButtonViewModel(style: .Done, action: .Submit, enabled: state.isValid),
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 102 characters (line_length)